src/main/java/com/yahoo/sketches/hash/MurmurHash3.java

```diff
@@ -250,7 +253,9 @@ void blockMix128(final long k1, final long k2) {
       h2 = finalMix64(h2);
       h1 += h2;
       h2 += h1;
-      return new long[] { h1, h2 };
+      state[0] = h1;
+      state[1] = h2;
+      return state;
```

Comment:
@hpx7 

When I did my performance characterization of this on May 3rd, (https://github.com/DataSketches/sketches-core/issues/200), I somehow missed this comment.  What you point out would be a problem even in a single-threaded environment.  Austin's CPP version has the user provide the result array via a pointer, which solves this problem.  We could do the same thing by forcing the user to provide the output array, but since there is no measurable performance advantage I don't see any reason to do this.